addons/website_slides/views/website_slides_templates.xml

```diff
@@ -108,12 +105,28 @@
                 <div class="float-left col-lg-8 col-md-12 col-12 mt16 mb8">
                     <h3 class="mt0">
                         <a t-attf-href="/slides/#{slug(channel)}" t-esc="channel.name"/>
+                        <span t-if="not channel.is_member" class="o_wslides_join_channel">
+                            <a role="button" class="btn btn-primary o_wslides_join_channel_link"
+                                title="Join Channel" aria-label="Join Channel" href="#"
+                                t-att-data-channel-id="channel.id">
+                                <i class="fa fa-chevron-right"></i> Join
```

Comment:
Not sure here but will the "Join" be translated even if it's not in a separated tag? (Like a <span>)

Reply:
I hope. Anyway those templates will change soon :) . It is just temporary for master.